testing/python/hexagon/diagnose_hmx.py

```
@@ -0,0 +1,144 @@
+import tilelang as tl
+import tilelang.language as T
+from tilelang.intrinsics.hexagon import hmx
+
+
+def has_hexagon_codegen():
+    try:
+        if not tvm.runtime.enabled("llvm"):
+            return False
+        tvm.target.Target("llvm -mtriple=hexagon -mcpu=hexagonv73")
+        return True
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)